app/controllers/register_api/applications_controller.rb

```diff
@@ -2,6 +2,29 @@ module RegisterAPI
   class ApplicationsController < ActionController::API
     include ServiceAPIUserAuthentication
 
-    def index; end
+    def index
+      application_choices = get_recruited_applications(
+        recruitment_cycle_year: recruitment_cycle_year_param,
+      )
+      render json: { data: MultipleApplicationsPresenter.new(application_choices, api: RegisterAPI).as_json }
+    end
+
+  private
+
+    def get_recruited_applications(recruitment_cycle_year:)
+      ApplicationChoice.includes([:application_form])
```

Comment:
When running the endpoint locally, this query is yielding n+1s due to further queries in the presenter. I think we might need to tweak it and the include list to ensure its more performant.